packages/cozy-intent/package.json

```diff
@@ -47,13 +48,11 @@
     "directory": "packages/cozy-intent"
   },
   "scripts": {
-    "build": "rm -rf ./dist && yarn build:types && babel --extensions .ts,.tsx --ignore 'src/tests','**/*.spec.tsx','**/*.spec.ts' ./src -d ./dist --copy-files --no-copy-ignored",
+    "build": "rm -rf ./dist && yarn build:types && babel --extensions .ts,.tsx --ignore '**/*.spec.tsx','**/*.spec.ts' ./src -d ./dist --copy-files --no-copy-ignored",
```

Comment:
let's split this command 🙏 

"build:clean": "rm -rf ./dist"

Reply:
why not using cozy-script?

Reply:
cozy-script is too heavy here I think, we don't really need webpack, just babel to transpile ts to js imo

Reply:
updated